meta-client/src/main/java/org/apache/helix/metaclient/factories/MetaClientConfig.java

```diff
@@ -123,6 +137,27 @@ public B setConnectionInitTimeoutInMillis(long timeout) {
       return self();
     }
 
+    /**
+     * Set timeout in ms for operation retry timeout
+     * @param timeout
+     * @return
+     */
+    public B setOperationRetryTimeoutInMillis(long timeout) {
+      _operationRetryTimeout = timeout;
+      return self();
+    }
+
+    /**
+     * Set reconnect policy when connection is lost or expired. By default is
+     * ExponentialBackoffReconnectPolicy
+     * @param reconnectPolicy an instance of type MetaClientReconnectPolicy
+     * @return
+     */
+    public B setMetaClientReconnectPolicy(MetaClientReconnectPolicy reconnectPolicy) {
```

Comment:
If we want to use default value, can we assign it at declaration time?

Reply:
If we want to assign at declaration time and user also set their own policy, we will be creating an default policy object and then throw away.